containers/gtk3/Dockerfile.fedora

```diff
@@ -0,0 +1,22 @@
+FROM docker.io/fedora:33 AS build-stage
```

Comment:
Why do you specify the `docker.io` (default) registry manually? I'm curious whether there is any kind of advantage in doing so.
Of course, it's fine, functionality wise.

Reply:
This is required with `buildah`, because the `docker.io` registry is not implicitly set by default.